tests/conftest.py

```diff
@@ -26,136 +28,61 @@
-    '/genomic-files',
-    '/cavatica-tasks',
-    '/cavatica-apps',
-    '/cavatica-task-genomic-files'
-]
-
-pytest_plugins = ['tests.mocks']
-
-
-@pytest.yield_fixture(scope='session')
-def app():
-    yield create_app('testing')
-
-
-@pytest.yield_fixture(scope='module')
-def client(app):
-    app_context = app.app_context()
-    app_context.push()
-    db.create_all()
-
-    mock = patch('dataservice.extensions.flask_indexd.requests')
-    mock = mock.start()
-    indexd_mock = MockIndexd()
-    mock.Session().get.side_effect = indexd_mock.get
-    mock.Session().post.side_effect = indexd_mock.post
-
-    mod = 'dataservice.api.study.models.requests'
-    mock_bs = patch(mod)
-    mock_bs = mock_bs.start()
-    
-    mock_resp_get = MagicMock()
-    mock_resp_get.status_code = 200
-    mock_resp_post = MagicMock()
-    mock_resp_post.status_code = 201
-
-    mock_bs.Session().get.side_effect = mock_resp_get
-    mock_bs.Session().post.side_effect = mock_resp_post
-
-    yield app.test_client()
-
-    mock_bs.stop()
-    mock.stop()
-    # Need to make sure we close all connections so pg won't lock tables
-    db.session.close()
-    db.drop_all()
-
-
-@pytest.yield_fixture(scope='function')
-def swagger(client):
-    yield json.loads(client.get('/swagger').data.decode('utf-8'))
-
-
-@pytest.fixture(scope='module')
-def entities(client):
-    """
-    Create mock entities
-    """
-    inputs = {
-        '/investigators': {
-            'external_id': 'inv001',
-            'name': 'submitter'
-        },
-        '/cavatica-task-genomic-files': {
-            'is_input': True
-        },
-        '/cavatica-apps': {
-            'external_cavatica_app_id': 'app1',
-            'name': 'MyApp',
-            'revision': 1,
-            'github_commit_url': 'http://www.github.com'
-        },
-        '/cavatica-tasks': {
-            'external_cavatica_task_id': str(uuid.uuid4()),
-            'name': 'MyTask'
-        },
-        '/genomic-files': {
-            'external_id': 'genfile001',
-            'file_name': 'hg38.fq',
-            'data_type': 'reads',
-            'file_format': 'fastq',
+ENTITY_TOTAL = 15
+SAMPLE_DATE = str(datetime.now().replace(tzinfo=tz.tzutc()))
+
+ENTITY_ENDPOINT_MAP = {
+    Study: '/studies',
+    Investigator: '/investigators',
+    StudyFile: '/study-files',
+    Family: '/families',
+    FamilyRelationship: '/family-relationships',
+    CavaticaApp: '/cavatica-apps',
+    SequencingCenter: '/sequencing-centers',
+    Participant: '/participants',
+    Diagnosis: '/diagnoses',
+    Phenotype: '/phenotypes',
+    Outcome: '/outcomes',
+    Biospecimen: '/biospecimens',
+    GenomicFile: '/genomic-files',
+    SequencingExperiment: '/sequencing-experiments',
+    CavaticaTask: '/cavatica-tasks',
+    CavaticaTaskGenomicFile: '/cavatica-task-genomic-files'
+}
+
+ENDPOINT_ENTITY_MAP = {v: k for k, v in ENTITY_ENDPOINT_MAP.items()}
+
+ENDPOINTS = list(ENTITY_ENDPOINT_MAP.values())
+
+ENTITY_PARAMS = {
```

Comment:
Can we move this to an external json file?

Reply:
But then we can't call functions to generate values (right now just uuids and dates, but might be more later)

Should I move all the constants into their own file like `tests/params.py`??

Reply:
It looks like the md5 hash is the only thing calling a function now? It could probably be hard-coded.